packages/sdk/src/utils/libp2p.ts

```diff
@@ -64,11 +64,16 @@ export async function defaultLibp2p(
     ? { metadata: wakuMetadata(shardInfo) }
     : {};
 
+  const filter =
+    typeof process !== "undefined" && process?.env?.NODE_ENV === "test"
```

Comment:
let's handle this the same way it happens here - https://github.com/waku-org/js-waku/blob/3d92f194a7d9f2803148da1ecc1d15ca8c5f0dae/packages/sdk/rollup.config.js#L25

this way we don't need to worry about process not being defined

Reply:
updated. tested it in light-js by making a release from this branch and made sure the replacement works